src/main/java/fluke/TaskList.java

```diff
@@ -0,0 +1,181 @@
+package fluke;
+
+import java.util.ArrayList;
+
+import fluke.exceptions.FlukeException;
+import fluke.exceptions.TaskDoesNotExistException;
+import fluke.tasks.Deadline;
+import fluke.tasks.Event;
+import fluke.tasks.Task;
+import fluke.tasks.Todo;
+
+/**
+ * TaskList is a wrapper class that manages the internal list of tasks by implementing
+ * various methods related to managing the list of tasks, such as adding and deleting.
+ */
+public class TaskList {
+    private final ArrayList<Task> listOfTasks;
```

Comment:
while ``listOfTasks`` is not wrong, might be better to rename to ``tasks`` instead due to this rule for coding standard.
![image](https://github.com/nus-cs2103-AY2324S1/ip/assets/40513197/e4dc686a-929f-4ad4-8664-4169a350bf0f)